screens/foodTypes.js

```diff
@@ -0,0 +1,133 @@
+import { useState } from "react";
+import { Image, View, Text, TouchableOpacity, Alert, FlatList, StyleSheet, } from "react-native";
+import { TextInput } from "react-native-web";
+import { useTailwind } from "tailwind-rn";
+ 
+
+
+const FoodTypes = () => {
```

Comment:
As this will a feature of the search bar, it makes more sense to add this as part of a search bar component rather than as a `screen`. The screens directory should be used for full screens or "pages" (larger features) rather than individual components (smaller  features).

Examples of screens include: Signup screen, Login Screen, Index Screen, Search results screen etc.

Additionally, since we only need to see the categories when a search is initiated, by default the categories should be hidden and then displayed when the user clicks on the search bar. See attached demo for an example taken from postmates.com

https://user-images.githubusercontent.com/18052272/215305542-1d3115d1-0f13-4c2a-90fc-93db7f8d958b.mp4

Reply:
I decided to leave it as a different screen so it would not interfere with anything that Mayank may do on Index.js. It also would make the length of Index.js a bit longer, which I wasn't sure if it would be okay.

I just saw these comments right now, I found a way for the categories to show up when the seatch bar is clicked, but man... I think tailwind would've made everything so much easier

Reply:
The length of the code does not matter as long as all of the code we need is there. With that said, if we make it part of the index page, the code for the index page will not get larger since we should make the search bar its own component and your search additions should go in that search component (not directly in the index page/screen).

It's also ok, if you want to make it its own page, but you will need to coordinate with @MayankTamakuwala because that will also change some of the implementation on the hope page. Additionally, we will also need to have a way to access the search screen you build. So if you decide to make its own page, you should also implement navigation. Here is some documentation on how you can achieve that:

https://reactnavigation.org/docs/bottom-tab-navigator

I would suggest using bottom style navigation tabs like the doordash mobile application does it. 

![doordashbar](https://user-images.githubusercontent.com/18052272/216225674-69233677-031d-4b5f-ab36-5e1aea0c4d84.png)

In our case we would need two tabs one for the index page and another for the search bar page.

Reply:
@dani5612 If you decide to go the single page route, you can mark this as resolved and merge your search screen (as long as the other issues are resolved), otherwise leave this thread unresolved until you finish the search bar component.